docs/source/release_notes/release-notes.rst

```diff
@@ -45,6 +45,18 @@ develop
          - Change
 
     *    - |new|
+         - Timelock clients now report tritium metrics for the ``TimestampService`` even if they are using the request batching service.
+           Note when setting up metric graphs, the timestamp service metrics are named with ``...Timelock.<getFreshTimestamp/getFreshTimestamps>`` when not using request batching,
+           but as ``...Timestamp.<getFreshTimestamp/getFreshTimestamps>`` if using request batching.
```

Comment:
This would be non-trivial, the name comes from the service you are instrumenting. When using request batching, we basically replace just the timestamp service (which wasn't instrumented in the past, but is now) and the timelock service continues to serve the locks.